example.md

```diff
@@ -0,0 +1,274 @@
+            xAxes: [{
+                scaleLabel: {
+                    display: true,
+                    labelString: 'day'
+                }
+            }]
+        }
+    }
+});
+```
+
+
+Bringing It All Together
+------------------------
+
+Finally, let's take all the parts and bring them together.
+For this, we can just copy most of the code we already have.
+
+The basic idea now is to:
+
+1. Request carbon monoxide data from the API using the JavaScript `fetch()` method
+   and preparing values and labels as a single list each.
+2. Create a chart and use these lists as data source for the chart.
+
+
+The resulting code should look somehwat like this:
+
+```js
+const api_url = 'https://demo.emissions-api.org/api/v1/average.json/'
+        + '?country=DE&begin=2019-02-01&end=2019-03-01'
+window.onload = function () {
+    fetch(api_url)
+    .then(response => response.json())
+    .then(data => {
+        let ctx = document.getElementById('average-example').getContext('2d');
+        new Chart(ctx, {
+            type: 'line',
+            data: {
+                // use start times contained in the requested data as labels
+                labels: data.map(x => x.start.substring(8, 10)),
+                datasets: [{
+                    label: 'Germany',
+                    backgroundColor: '#93bd20',
+                    // use the average values as data
+                    data: data.map(x => x.average),
+                }]
+            },
+
+            // add a few sensible configuration options
+            options: {
+                scales: {
+                    yAxes: [{
+                        scaleLabel: {
+                            display: true,
+                            labelString: 'carbon monoxide [mol/m²]'
+                        },
+                        ticks: {
+                            beginAtZero: true
+                        }
+                    }],
+                    xAxes: [{
+                        scaleLabel: {
+                            display: true,
+                            labelString: 'day'
+                        }
+                    }]
+                }
+            }
+        });
+    })
+}
+```
+
+That's it!
+
+
+
+<script src="https://cdn.jsdelivr.net/npm/chart.js@2.9.2/dist/Chart.min.js"></script>
+<script>
+const api_url = 'https://demo.emissions-api.org/api/v1/average.json/'
+        + '?country=DE&begin=2019-02-01&end=2019-03-01'
+window.onload = function () {
+    fetch(api_url)
+    .then(response => response.json())
+    .then(data => {
+        let ctx = document.getElementById('average-example').getContext('2d');
+        new Chart(ctx, {
+            // The type of chart we want to create
+            type: 'line',
+
+            // The data for our dataset
+            data: {
+                labels: data.map(x => x.start.substring(8, 10)),
+                datasets: [{
+                    label: 'Germany',
+                    backgroundColor: '#93bd20',
```

Comment:
Let me be nit-picky here: I do not like the use of a green background for the area under the curve. It messes up the [data-ink ratio](https://infovis-wiki.net/wiki/Data-Ink_Ratio).

Reply:
The green background is the data-ink, isn't it? It's a representation of the actual data. It's like saying use a pie or ring chart, but do not fill the parts with color. That's possible of course but actually goes against the data-ink ratio goals you linked. After all, you are taking away ink representing the data.

Reply:
The information is the data point defined by an x/y coordinate. From an information design point of view, you would only fill the background if you want the audience to actually be able to compare relative areas (but this graph only shows one kind of measurement). What we are only able to assess from this graph are relative differences between values over time that are best represented by dots (or connected dots, if you want to stress the time-dependency between the values). 

And don't even get me started on pie or ring charts...